src/components/views/rooms/EventTile.tsx

```diff
@@ -1348,9 +1348,9 @@ export class UnwrappedEventTile extends React.Component<EventTileProps, IState>
                         <a href={permalink} onClick={this.onPermalinkClicked}>
                             { timestamp }
                         </a>
-                        { msgOption }
                     </div>,
                     reactionsRow,
+                    msgOption,
```

Comment:
I note that the non-threads case does:

```
React.createElement("li", <attrs>, 
<>
  { elem }
  { elem }
</>);
```

whereas this bit does

```
React.createElement("li", <attrs>, 
[
  elem,
  elem,
]);
```

Is there a difference? or any reason to prefer one style over the other?